common/main/kxv.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<ldml>
+	<identity>
+		<version number="$Revision$"/>
+		<language type="kxv"/>
+	</identity>
+	<characters>
+		<exemplarCharacters>[a ā {aa} {āā} i ī u ū e ē o ō k g ṅ c j ñ ṭ ḍ ṇ t d n p b m r ḷ v s h y l ṛ ◌̃ ◌̄]</exemplarCharacters>
```

Comment:
```suggestion
		<exemplarCharacters>[a ā {aa} {āā} i ī u ū e ē o ō k g ṅ c j ñ ṭ ḍ ṇ t d n p b m r ḷ v s h y l ṛ]</exemplarCharacters>
```

These can't go in exemplars per se:
```
̃  U+0303 | COMBINING TILDE
̄  U+0304 | COMBINING MACRON
```

You could add for example `{k̃} {t̄}` etc— is it known which characters take these?